File: C#/rest-service/Models/Games.cs
public class Games {
    public Games(string name, string owner, string releaseYear) {   
        this.Name = name;
        this.Owner = owner;
        this.ReleaseYear = releaseYear;
    }

    public string Name {get; set;}
    public string Owner {get; set;}
    public string ReleaseYear {get; set;}

    override
    public string ToString() {
        return "Name: " + Name + "; Owner: " + Owner + "; ReleaseYear: " + ReleaseYear + ";";
    }
}

public static class AllGames {
    public static List<Games> listGames = new List<Games>();
}

File: Python/rest-service/model/Games.py
class Games:
    def __init__(self, name, owner, release_year):
        self.name = name
        self.owner = owner
        self.release_year = release_year
        
    def __str__(self):
        return "Name: " + self.name + "; Owner: " + self.owner + "; ReleaseYear: " + self.release_year + ";"


class AllGames:
    list_games = []
